.github/workflows/release.yml

```
@@ -178,6 +174,21 @@ jobs:
         echo "Created tag ${TAG}"
         echo "release-needed=true" >> $GITHUB_OUTPUT
 
+    - name: Create pull request with regenerated llms.txt
+      if: steps.tag.outputs.release-needed == 'true'
+      uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.12)

[error] the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)